contribution.md

```diff
@@ -0,0 +1,237 @@
+# How to contribute to OMZ
+
+From this document you will know how to contribute your model to OpenVINO&trade; Open Model Zoo. Almost any model from supported frameworks (see list below) can be added. To do this do next few steps.
+
+1. [Model location](#model-location)
+2. [Model conversion](#model-conversion)
+3. [Demo](#demo)
+4. [Accuracy validation](#accuracy-validation)
+7. [Configuration file](#configuration-file)
+6. [Documentation](#documentation)
+7. [Pull request requirements](#pull-request-requirements)
+
+List of supported frameworks:
+* Caffe\*
+* Caffe2\* (by conversion to ONNX\*)
+* TensorFlow\*
+* MXNet\*
+* PyTorch\* (by conversion to ONNX\*)
+
+
+
+## Model location
+
+Upload your model to any Internet file storage with easy and direct access to it. It can be www.github.com, GoogleDrive\*, or any other.
+
+*After this step you will get **links** to the model*
+
+## Model conversion
+
+OpenVINO&trade; supports models in its own format IR. Model from any supported framework can be easily converted to IR using Model Optimizer tool included in OpenVINO&trade; package. More information about conversion you can learn [here](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html). After successful conversion you will get model in IR format `*.xml` representing net graph and `*.bin` containing net parameters. 
+
+> **NOTE 1**: due OpenVINO&trade; paradigms, mean and scale values are built-in converted model.
```

Comment:
might be re-prhase like: due to OpenVINO paradigm, image pre-processing parameters (mean and scale) should be built into converted model to simplify model usage.